packages/presets/src/blocks/ai-chat-block/ai-chat-model.ts

```diff
@@ -8,7 +8,9 @@ type AIChatProps = {
   index: string;
   scale: number;
   messages: string; // JSON string of ChatMessage[]
-  sessionId: string;
+  sessionId: string; // forked session id
+  rootWorkspaceId: string; // workspace id of root chat session
+  rootDocId: string; // doc id of root chat session
```

Comment:
Why do we need this two property?

Reply:
The AI Chat block can be copied and pasted in different doc, and should support chatting in different doc. When chatting and querying the forked session histories, when calling the backend api, the root session doc id and workspace id is needed. 

related issue